packages/server/src/DataSource.ts

```diff
@@ -51,18 +52,7 @@ export const init = async (): Promise<void> => {
                 username: process.env.DATABASE_USER,
                 password: process.env.DATABASE_PASSWORD,
                 database: process.env.DATABASE_NAME,
-                ...(process.env.DATABASE_SSL_KEY_BASE64
-                    ? {
-                          ssl: {
-                              rejectUnauthorized: false,
-                              cert: Buffer.from(process.env.DATABASE_SSL_KEY_BASE64, 'base64')
-                          }
-                      }
-                    : process.env.DATABASE_SSL === 'true'
-                    ? {
-                          ssl: true
-                      }
-                    : {}),
+                ssl: getDatabaseSSLFromEnv(),
```

Comment:
This will result in breaking changes for existing Flowise pg installs given the way pg node uses env vars internally on connection instantiation.

Using ssl: getDatabaseSSLFromEnv() will result in ssl: {} when there are no Flowise env vars present. That will in turn not allow the usage of PGSSLMODE, which will be a breaking change for existing Flowise installs using pg. 

getDatabaseSSLFromEnv should be spread instead of being a fixed value for ssl so that pg node continues to function as is. In turn getDatabaseSSLFromEnv should return ssl as the top level object when the Flowise env vars are present.

Reply:
@Jaredude what if we'll return undefined in case no env vars present?

Reply:
Yes, returning undefined instead of {} will work 
<img width="647" alt="Screenshot 2024-01-30 at 3 34 13 PM" src="https://github.com/FlowiseAI/Flowise/assets/10228950/d71b4ae1-c8d8-4982-b2e0-50bcf2dd4d2a">